catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -2146,6 +2146,9 @@ private QueryResponse doQuery(QueryRequest queryRequest, FederationStrategy stra
 
         LOGGER.debug("Calling strategy.federate()");
 
+        if (fanoutEnabled) {
+            queryRequest.getProperties().put("no-default-tags", true);
```

Comment:
We probably should have a discussion on this. The problem is that it is supposed to get replaced (only supposed to be a temporary fix..). maybe we should discuss with core-api team? @pklinef